webcomponents/bundle/src/main/bundle/search.properties

```
@@ -1,8 +1,11 @@
 search_tooltip=Search your content
+search_placeholder=Enter search term to search Sakai
+search_min_length=Please enter at least {} characters
+search_min_length_value=3
 site_label=Site: 
 search_result_title=Title:
 toolname_announcement=Announcement
-toolname_assignment=Assignment
+toolname_assignment=Assignment 
```

```suggestion
toolname_assignment=Assignment
```
I think we can remove that?